lib/durationPicker.dart

```diff
@@ -0,0 +1,649 @@
+// Code taken and modified from flutter duration picker package
```

Comment:
Since this file is almost entirely code that is not ours, I would add a link to the package or the original file being pasted so it can be referenced by others if necessary. It also wouldn't hurt to add a comment up here to briefly explain why we're modifying the package instead of using it as-is.